### PR TITLE
Add overview document

### PR DESCRIPTION
**Pull request by RoyJacobs**
An initial attempt to write down (in very high-level terms) what we're trying to achieve.

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Overview
+
+## Goal
+The goal of Blueprint is to be an API registry, with the following initial goals:
+
+1. Provide a basic registry for Swagger 2 schemas
+1. Provide a basic registry for JSON schemas
```
[the remark this answers is not in the document]
Comment (mzeijen) [reply docs/overview.md]: Am these are the initial goals and what I describe are the future goals :)

[the remark this answers is not in the document]
Comment (RoyJacobs) [reply docs/overview.md]: Indeed :)

File: docs/overview.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+1. Namespace
+1. Schema name
+1. Version
+1. Artifact(s)
+
+#### Schema
+A schema also defines a `type`, which defines:
+1. Which format is the schema (Swagger 2, JSON schema, OpenAPI 3, etc)
+1. What kind of versioning does it use (Maven, npm, etc)
```
[the remark this answers is not in the document]
Comment (RoyJacobs) [reply docs/overview.md]: Yes, indeed. Will clarify.

File: docs/overview.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mzeijen) [reply docs/overview.md]: LGTM

**Pull request merged**